onnxruntime/core/session/inference_session.cc

```diff
@@ -102,8 +102,10 @@ InferenceSession::InferenceSession(const SessionOptions& session_options,
     : session_options_(session_options),
       graph_transformation_mgr_(session_options.max_num_graph_transformation_steps),
       logging_manager_(logging_manager),
+#ifdef USE_OPENMP
```

Comment:
Do you mean the other way around? It should be #ifndef, right?

Reply:
you are right

Reply:
else else else !!!!!!